src/GraphQL/Types/Scalars/ScalarGraphType.cs

```diff
@@ -152,5 +165,25 @@ public virtual IValue ToAST(object value)
                 return null;
             }
         }
+
+        protected internal object ThrowLiteralConversionError(IValue input)
+        {
+            throw new ArgumentException($"Unable to convert '{input}' to '{Name}'");
+        }
+
+        protected internal IValue ThrowASTConversionError(object value)
+        {
+            throw new InvalidOperationException($"Unable to convert '{value}' of the scalar type '{Name}' to an AST representation.");
+        }
+
+        protected internal object ThrowValueConversionError(object value)
+        {
+            throw new ArgumentException($"Unable to convert '{value}' to '{Name}'");
+        }
+
+        protected internal object ThrowSerializationError(object value)
+        {
+            throw new InvalidOperationException($"Unable to serialize '{value}' to '{Name}'.");
```

Comment:
```suggestion
            throw new InvalidOperationException($"Unable to serialize '{value}' to the scalar type '{Name}'.");
```

Reply:
👍

Once I can get existing tests to pass again, then I'll commit the revised exception messages.